daemon/cluster/cluster.go

```diff
@@ -270,6 +270,16 @@ func (c *Cluster) GetAdvertiseAddress() string {
 	return c.currentNodeState().actualLocalAddr
 }
 
+// GetDataPathAddress returns the address to be used for the data path traffic.
```

Comment:
Would you prefer something like: GetDataPathAddress returns the address to be used for the data path traffic, if specified.